Laba2SCCS/src/DataAccessLayer/DTO/InspectionContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.DTO
```

Comment:
It shouldn't be in DTO folder.
Lets it will be on the root of DataAccessLayer folder